prisma/schema.prisma

```diff
@@ -11,14 +11,111 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Log {
-  id      String @id @default(uuid())
-  level   Level
-  message String
+model Ship {
+  id      String  @id @default(uuid())
+  name    String  @unique
+  // maneuvers Maneuver[] // I don't know how we want to handle custom types here, because what I want is a {ManeueverType, Difficulty} tuple, e.g. {"5K", "Red"}
+  // If we ever hit the case of a chassis having different hull/shields (1.0 YT-1300), we'd have to move these to the pilot
+  hull    Int
+  shields Int
+  // arcs Arc[] // Same as maneuevers - I want tuples and I don't know the right way to do it with typing, e.g. [{"front", "2'}, {"bullseye", "3'}]
+  pilots  Pilot[]
 }
 
-enum Level {
-  Info
-  Warn
-  Error
+model ShipAbility {
```

Comment:
May as well put the comment here actually - Yes, the rules reference describes ship abilities as separate from pilot abilities, but I'm not sure that it's an actually useful description of reality.

Is Pursuit Craft really a ship ability? Networked Calculations?

Reply:
The main thing I want to achieve here is having a single place where I can reference e.g. "Autothrusters" without repeating the text of it on every pilot that currently has it. I recognize that this makes the assumption that there won't be a pilot that as an "Authothrusters" ability that's DIFFERENT from the ones that are currently printed, but... that feels like a safe assumption?

One option is to pull ALL abilities into something like this... but that feels like overkill for all of the abilities that are on a single pilot... hmm.

Reply:
Yeah, it's just kind of an ugly solution, and to be clear I don't have any better ideas, haha. Just one of those things where the game rules don't map to the actual situation at hand very well.

Reply:
Maybe a `SharedAbility` model?

Reply:
Yeah, something like that? To make matters worse we now have unique pilots without abilities, from gauntlet pilots in each faction. And, some ship abilities aren't unique, like Rigged to Blow/Co-Pilot (escape craft). Honestly I think your current solution is probably the best one already? The game rules refer to pilot/ship abilities, it's how people conceptualize it regardless, and the number of exceptions we're talking about is so small that I think I'm leading you off course!